variants/VAKE_F446VE/ldscript.ld

```diff
@@ -0,0 +1,169 @@
+/*
+*****************************************************************************
+**
+
+**  File        : LinkerScript.ld
+**
+**  Abstract    : Linker script for STM32F446VETx Device with
+**                512KByte FLASH, 128KByte RAM
+**
+**                Set heap size, stack size and stack location according
+**                to application requirements.
+**
+**                Set memory bank area and size if external memory is used.
+**
+**  Target      : STMicroelectronics STM32
+**
+**
+**  Distribution: The file is distributed as is, without any warranty
+**                of any kind.
+**
+**  (c)Copyright Ac6.
```

Comment:
This licensing is not correct.
See https://github.com/stm32duino/Arduino_Core_STM32/commit/e7af82b63ecf7affbe5dd19eacceb43bb43019fc

Reply:
updated